api/src/pcapi/routes/native/v1/serialization/bookings.py

```diff
@@ -54,8 +58,47 @@ class BookingOfferExtraData(BaseModel):
     ean: str | None
 
 
+class BookingOfferResponseAddress(BaseModel):
+    street: str | None
+    postalCode: str | None
+    city: str | None
+    label: str | None
+    coordinates: Coordinates
+    timezone: str
+
+    class Config:
+        orm_mode = True
+
+
+class BookingOfferResponseGetterDict(GetterDict):
+    def get(self, key: str, default: Any | None = None) -> Any:
+        if key == "address":
+            offerer_address: OffererAddress | None
+            if not self._obj.offererAddress:
+                offerer_address = self._obj.venue.offererAddress
+            else:
+                offerer_address = self._obj.offererAddress
```

Comment:
Suugestion : Plus facile à lire et évitant le "not not" : 
```suggestion
            if self._obj.offererAddress:
                offerer_address = self._obj.offererAddress
            else:
                offerer_address = self._obj.venue.offererAddress
```